### ISSUE TITLE
Calendar Illustration

### ISSUE THREAD

**Issue by duelj** [cfpb/design-manual#43]
Here were some different options for the calendar illustration that I put together. It's first use will be in Mollie's Ideas 42 video so I am not sure if there will be specific needs because of that or not. 
![cal_1](https://f.cloud.github.com/assets/6873734/2398034/61e9a304-a9f1-11e3-86d8-8919da05606d.png)
![cal_3](https://f.cloud.github.com/assets/6873734/2398035/61ef3292-a9f1-11e3-8913-a754b6fa046d.png)
![cal_2](https://f.cloud.github.com/assets/6873734/2398036/61f0b1ee-a9f1-11e3-9432-caad0d275fad.png)
![cal_4](https://f.cloud.github.com/assets/6873734/2398037/61f2af08-a9f1-11e3-8db2-33954326424f.png)

**Comment by mebates** [comment]
Nice! Thanks @duelj 

The second one is best for the video I'm working on. I need the days to fill with $$ signs, so it's important to have that grid. I like the folded corner. Since this is an illustration that will probably be used at a larger size, do you think we can fit the full month name in slightly smaller type?

**Comment by duelj** [comment]
Yea sounds good, maybe something like this would work? 
![cal_5](https://f.cloud.github.com/assets/6873734/2399517/5420a6ec-a9ff-11e3-8542-d0a4e96b88d0.png)

**Comment by stephanieosan** [comment]
:+1:

**Comment by mebates** [comment]
Nice. Do you think we need to have 7 days in the grid to make it accurate? And 5 weeks deep? I would make the red field slightly narrower, top to bottom.

**Comment by duelj** [comment]
Maybe something like this? 
![cal_6](https://f.cloud.github.com/assets/6873734/2400424/6cd17832-aa0a-11e3-820b-3a220442b889.png)

**Comment by nataliafitzgerald** [comment]
Independent of the ideas 42 project I like the calendar that is 3rd on the list and says "MAR 14." This would make a good date calendar (the type that sits on a desk that you flip each day). 

In the last option, for ideas42, does this illustration represent a large, flat desk calendar or a hanging wall calendar? If this is a large, flat desk calendar perhaps we remove the hooks and add a bit of dimension/shadow to the bottom of the top bar to indicate that this is a tear off calendar and there is some depth here (along the bottom of the red box). Is there a way to simplify the calendar page, maybe by using uniform stroke weights, or by making the dollar signs dark gray rather than green, or by using filled boxes to indicate the days of the week rather than a line grid?

**Comment by nataliafitzgerald** [comment]
Following up on my last comment, if this is a tear off/ desk pad calendar, the month would appear on the calendar sheet, rather than the top part (red bar). 

For comparison, we could also try a spiral alternative for this one.

**Comment by ohsk** [comment]
I like the general directions of #2 & #3 from the original set! 

I agree with Natalia that in the most recent directions, the stroke weights of the daily boxes is what's throwing me off and would be great to see what a filled version would look like.

I think it makes sense to keep the month name on the red bar; less concerned about how this maps conceptually to a wall calendar or desk pad calendar. It has a strong visual effect with that color bar.

**Comment by duelj** [comment]
That makes sense with the boxes, they are kinda chaotic, good call it feels much better. I tried a couple of options again to look at. I tried a little extra detail on the tear off page version, but I im not sure it adds enough to keep. 
![cal_7](https://f.cloud.github.com/assets/6873734/2402362/271c30c2-aa21-11e3-8d85-382ef5f4f314.png)
![cal_8](https://f.cloud.github.com/assets/6873734/2402363/271c7b86-aa21-11e3-98a3-bb9afe5d6e2f.png)
![cal_9](https://f.cloud.github.com/assets/6873734/2402364/271ca52a-aa21-11e3-9044-56ccca1921ff.png)
![cal_9a](https://f.cloud.github.com/assets/6873734/2402366/271d132a-aa21-11e3-833f-b321f889ee2e.png)
![cal_10](https://f.cloud.github.com/assets/6873734/2402365/271d0b32-aa21-11e3-9fbb-c4a2c95bfcfa.png)

**Comment by mebates** [comment]
Hey @duelj where did we land on this? Can we pick one and close the issue? I prefer the third one down, with slightly smaller dollar signs and a smaller corner.

**Comment by duelj** [comment]
I believe this is what people were looking for?
- Make the shadow not cover the top red bar (I think this was because the red bear is separate from the tear-off style calendar page?)
- got rid of the tear detailing
- tried Stephen's corner fold style with the stroke. Having some color issues with the cast shadow and how to handle the fold. 
- removed some boxes to make it feel like a real calendar where the month ends mid week, etc. 
- scaled down the dollar signs (couldn't remember if these were staying) I thought they were needed for the video? 

![cal_11](https://f.cloud.github.com/assets/6873734/2440321/ba45b0d8-ae06-11e3-9a3f-670edbeee12b.png)

...and I think people generally seemed OK with this for another single date style calendar maybe? The corner fold will likely need updating with what ever final tweaks end up getting made to how we handle that. 
![cal_2](https://f.cloud.github.com/assets/6873734/2440357/eda0ade8-ae06-11e3-975c-13cf21da9c94.png)

**Comment by nataliafitzgerald** [comment]
I prefer this version for the video we are working on now. 

The edit that was suggested at the design crit was to remove the last row of squares and to make the dollar signs slightly smaller so that they don't come so close to the edges of the squares. Can we make the gray of the fold one shade lighter?  

![screen shot 2014-03-17 at 3 25 13 pm](https://f.cloud.github.com/assets/6887128/2440652/f15b65b0-ae09-11e3-8f2e-d4c88fa69681.png)

**Comment by duelj** [comment]
Here are the changes you requested:
![cal_12](https://f.cloud.github.com/assets/6873734/2440760/71a844e4-ae0b-11e3-842e-53c67aba4233.png)

**Comment by nataliafitzgerald** [comment]
I worked on the calendar a bit more. Seeing the lighter shadow (above) I realize that it doesn't provide enough contrast with the date boxes underneath. So, I went back to a slightly darker fold. I also adjusted the kerning of the word "March." 
![calendar_1_calendar](https://f.cloud.github.com/assets/6887128/2453004/e76d2190-aed8-11e3-83b7-ebe2e02d0816.png)

I also tried a version without the fold. Maybe it isn't needed?
![calendar_2_calendar](https://f.cloud.github.com/assets/6887128/2453006/f0b45886-aed8-11e3-80de-23101867b6f9.png)
